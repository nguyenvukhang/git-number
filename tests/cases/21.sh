# change cwd mid-way

init 3
# create and commit ./src
mkdir src
touch src/.gitkeep
_gitnu add src
_gitnu commit -m "src_dir"

# main test
pushd src >/dev/null
touch emerald sapphire ruby
_gitnu status # ran from /src
popd
_gitnu add 3-5 # ran from /
log gitnu status
