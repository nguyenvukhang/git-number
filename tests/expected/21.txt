On branch main
Changes to be committed:
1	[32mnew file:   file_0003[m
2	[32mnew file:   src/emerald[m
3	[32mnew file:   src/ruby[m

Untracked files:
4	[31mfile_0001[m
5	[31mfile_0002[m
6	[31msrc/sapphire[m

